live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/messaging/hx_insight/storage/connector/HttpFileUploader.java

```diff
@@ -123,7 +127,7 @@ STORAGE_LOCATION_HEADER, wrapRawToken(fileUploadRequest.storageLocation()),
             }
             catch (IOException ioe)
             {
-                log.atDebug().log("Upload :: Stream reset failed due to: {}", ioe.getMessage());
+                log.warn("Storage :: Rendition stream NOT reset properly after node %s content upload fail due to: %s".formatted(nodeId, ioe.getMessage()), ioe);
```

Comment:
Similarly I think we want `atWarn()` here.

Reply:
Fixed - thanks!